clean.go

```diff
@@ -32,19 +32,14 @@ type cleanCmd struct {
 }
 
 func newCleanCmd() *cleanCmd {
-	h := &intHeap{}
-	heap.Init(h)
-
 	return &cleanCmd{
 		dlPkgName:   "dl", // default package name
-		removedIdxs: h,
+		removedIdxs: newintHeap(nil),
 		astFile:     nil,
 	}
 }
 
-var (
-	excludedFiles = []string{dlDir, ".git"}
-)
+var excludedFiles = []string{dlDir, ".git"}
```

Comment:
[ask] Is this thread safe?

I think this is okay because `$ go test -race` is passed.